credentials/credentials_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func TestGetLicenseKeySecretId(t *testing.T) {
 	secretId := getLicenseKeySecretId(&config.Configuration{})
-	assert.Equal(t, defaultSecretId, secretId)
+	assert.Equal(t, "", secretId)
```

Comment:
Looks like more tests are needed here too?

Reply:
This is supposed to return an empty string if there's no secret configured.

Reply:
Yeah, I was just making sure you added more tests, like you have below. 👍

Reply:
Oh gotcha, I can add a few more